compat/src/index.js

```diff
@@ -71,9 +71,14 @@ class ContextProvider {
  */
 function Portal(props) {
 	let wrap = h(ContextProvider, { context: this.context }, props.vnode);
-	render(wrap, props.container);
+	let container = props.container;
+	container._prevVNode = null;
```

Comment:
Could `hydrate` be used here instead? `hydrate` does this inside it. So, we could end-up with:
```suggestion
	let temp = props.container.appendChild(document.createTextNode(''));
	hydrate(wrap, container);
```

I guess this would compress better, maybe? I couldn't verify it atm.